.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can we please remove this?

Reply:
Sure. I'll add some adidtional `.gitignore` rulres.